Makefile

```
@@ -53,7 +54,19 @@ pull_translations:  ## pull translations from Transifex
 	git clean -fdX conf/locale/eo
 	i18n_tool validate --verbose
 	paver i18n_compilejs
-
+else
+pull_translations:  ## pull translations from Transifex
+	find conf/locale -mindepth 1 -maxdepth 1 -type d -exec rm -r {} \;
+	atlas pull $(OPENEDX_ATLAS_ARGS) translations/edx-platform/edx-platform/conf/locale:conf/locale
+	$(MAKE) extract_translations
+	i18n_tool dummy
+	i18n_tool generate --verbose 1
+	git clean -fdX conf/locale/rtl
+	git clean -fdX conf/locale/eo
+	i18n_tool validate --verbose
```

@OmarIthawi , do we need all steps when pulling using Atlas? or we should keep only generate?

```suggestion
	i18n_tool generate --verbose 1
	i18n_tool validate --verbose
```